places-solution-integrations/campaign/README.md

```diff
@@ -0,0 +1,5 @@
+# Places Integration with Adobe Campaign Standard
+
+Below is the list of current types of integration between Adobe Places and Adobe Campaign Standard:
```

Comment:
Change to:
"Here is the current integration type between Adobe Places and Adobe Campaign Standard:"